### ISSUE TITLE
conda-forge CI failing on Linux and Windows June 2024

### ISSUE THREAD

**Issue by traversaro** [robotology/robotology-superbuild#1666]
~~~
2024-06-20T02:40:42.3658429Z -- Could NOT find Qhull (missing: Qhull_DIR)
2024-06-20T02:40:42.3659161Z -- Could NOT find Qhull (missing: Qhull_DIR)
2024-06-20T02:40:42.3660705Z CMake Error at /home/runner/miniconda3/envs/test/share/cmake-3.28/Modules/FindPackageHandleStandardArgs.cmake:230 (message):
2024-06-20T02:40:42.3662238Z   Could NOT find Qhull (missing: qhull_INCLUDE_DIR qhull_r_LIBRARY)
2024-06-20T02:40:42.3663004Z Call Stack (most recent call first):
2024-06-20T02:40:42.3664722Z   /home/runner/miniconda3/envs/test/share/cmake-3.28/Modules/FindPackageHandleStandardArgs.cmake:600 (_FPHSA_FAILURE_MESSAGE)
2024-06-20T02:40:42.3666241Z   cmake/FindQhull.cmake:29 (find_package_handle_standard_args)
2024-06-20T02:40:42.3667299Z   cmake/BipedalLocomotionFrameworkFindDependencies.cmake:58 (find_package)
2024-06-20T02:40:42.3668757Z   cmake/BipedalLocomotionFrameworkDependencies.cmake:41 (checkandset_dependency)
2024-06-20T02:40:42.3669730Z   CMakeLists.txt:21 (include)
2024-06-20T02:40:42.3670078Z 
2024-06-20T02:40:42.3670085Z 
2024-06-20T02:40:42.3670416Z -- Configuring incomplete, errors occurred!
~~~

**Comment by traversaro** [comment]
Last working: https://github.com/robotology/robotology-superbuild/actions/runs/9492581215
First failing: https://github.com/robotology/robotology-superbuild/actions/runs/9509563175

**Comment by traversaro** [comment]
Apparently in https://github.com/robotology/robotology-superbuild/pull/1275 we switched to use qhull from conda-forge, but we never actually explicitly installed qhull, we just relied on its being installed as a dependency of pcl. This is a problem, but what is happening is that for some conflict/constraint now pcl 1.8.1 is being installed. Hopefully this will be solved by https://github.com/conda-forge/pcl-feedstock, but in general it could make sense to have some lower boundary on pcl to avoid confusing failures.
